config/reader.go

```diff
@@ -85,6 +85,29 @@ func getCloudCacheFilePath(id string) string {
 	return filepath.Join(ViamDotDir, fmt.Sprintf("cached_cloud_config_%s.json", id))
 }
 
+type cachedTLSConfig struct {
+	Certificate string `json:"tls_certificate"`
+	PrivateKey  string `json:"tls_private_key"`
+}
+
+func readTLSFromCache(id string) (*cachedTLSConfig, error) {
```

Comment:
could we use readFromCache and then only grab the TLS section out of it? code seems pretty similar

Reply:
it looks like that function expects json with the full cloud config object? let me try though